luis-eduardo-salcedo/Tarea-11/router/src/pages/taskFive.js

```diff
@@ -0,0 +1,13 @@
+import ContentTask from '../components/contentTask';
+import Nav from '../components/navbar';
+import taskFiveImg from '../img/taskFive.png';
+
+export default function taskThree() {
+    return (
```

Comment:
@luisSalcedo8 , this is a very bad approach there should be only one Task component and use props  to pass information